### PR TITLE
Switch to SLE15SP1 for deployer node (SOC-9680)

### PR DESCRIPTION
**Pull request by dirkmueller**
Based on #628 however including the official repositories instead.

### CODE REVIEW

File: vars/deploy-on-openstack.yml [hunk 1]
```diff
@@ -30,6 +30,14 @@ deploy_on_openstack_osh_repos_per_imagename:
     - openSUSE-Leap-15.0-Cloud
     - Leap15develtools
     - socok8s
+  SLE-15-SP1-JeOS-GM:
```
Comment (evrardjp) [inline vars/deploy-on-openstack.yml]: Could you add a ```#TODO: remove SLES-15-SP1-openstack-master when caasp3 compat is removed```, please?

Comment (dirkmueller) [reply vars/deploy-on-openstack.yml]: @evrardjp I don't see it in the list of repositories, why should I add a comment about a repository that isn't used? ;) I guess I should squad the commits at some point..

Comment (evrardjp) [reply vars/deploy-on-openstack.yml]: That was just before you pushed a patch I think or because I am blind. Just ignore this. :)

**Comment by evrardjp** [comment]
Approved the concept, waiting for ci.

**Comment by jsuchome** [comment]
`Problem: nothing provides libcrypto.so.1.0.0()(64bit) needed by phantomjs-2.1.1.0-1.1.x86_64`

**Comment by evrardjp** [comment]
> `Problem: nothing provides libcrypto.so.1.0.0()(64bit) needed by phantomjs-2.1.1.0-1.1.x86_64`

phantomjs will go away in caasp4, it makes me very sad to have to fix this just before ripping this out.

**Comment by dirkmueller** [comment]
@evrardjp legacy module, added now.

**Comment by evrardjp** [comment]
That looks better. Respinning CI in my own PR.

**Comment by evrardjp** [comment]
Looks okay to me, if it passes CI.

**Comment by evrardjp** [comment]
Should we remove python3-cmd2? Not sure it brings something here.

**Comment by jsuchome** [comment]
`Package '+python-pyOpenSSL' not found`

**Comment by evrardjp** [comment]
This CI job was stuck for an hour and a half without doing anything, so I stopped it.

**Comment by evrardjp** [comment]
Ok the restart seems better, I can see logs now. Weird.

**Pull request merged**